src/Messages/TSMessagesManager.h

```diff
@@ -20,14 +21,6 @@ NS_ASSUME_NONNULL_BEGIN
 @interface TSMessagesManager : NSObject
 
 - (instancetype)init NS_UNAVAILABLE;
-
-- (instancetype)initWithNetworkManager:(TSNetworkManager *)networkManager
```

Comment:
To me that doesn't seem any safer than *not* including the headers outside of tests at all, so I'm gong to leave it as is for now.